styles/style.css

```diff
@@ -1,8 +1,181 @@
-/*
-Colors:
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap');
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+:root {
+    --color-text: #1a1a1a;
+    --color-green: #00b172;
+    --color-white: #fff;
+}
```

Comment:
Good use of setting the base colors for a website. CSS variables a good way to organize your work ✅